lib/jerboa/client.ex

```diff
@@ -65,7 +65,7 @@ defmodule Jerboa.Client do
 
   alias Jerboa.Client
 
-  @doc """
+  @doc ~S"""
```

Comment:
Credo... It complains that there are multiple quote characters inside a string, so it's better to use `~S` sigil which creates a raw string. Credo can't understand that it's a docstring which already is a raw string..